README.md

```diff
@@ -1,19 +1,14 @@
 # WordPress for iOS #
 
-[![Build status](https://badge.buildkite.com/2f3fbb17bfbb5bba508efd80f1ea8d640db5ca2465a516a457.svg)](https://buildkite.com/automattic/wordpress-ios)
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
-
 ## Build Instructions
 
 Please refer to the sections below for more detailed information. The instructions assume the work is performed from a command line.
 
-> Please note – these setup instructions only apply to Intel-based machines. M1-based Mac support is coming, but isn't yet supported by our tooling.
-
 ### Getting Started
 
-1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. *WordPress for iOS* requires Xcode 11.2.1 or newer.
-1. From a command line, run `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
-1. Now, run `cd WordPress-iOS` to enter the working directory.
+1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. *WordPress for iOS* requires Xcode `14.3.1` or newer.
```

Comment:
A spelled out Xcode version in the README is bound to become out of sync, like the diff shows.

What do you think of mentioning the `.xcode-version` file?

https://github.com/wordpress-mobile/WordPress-iOS/blob/05a954e50ad82c1e651ce0e972bcb8dadc5c7be6/.xcode-version

```suggestion
1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. Refer to the `.xcode-version` file for the minimum required version.
```

Reply:
Good call! One less thing to maintain. 👍